api/users.go

```diff
@@ -36,10 +39,11 @@ func GetUsers(c *gin.Context) {
 func GetUser(c *gin.Context) {
 	user, err := store.GetUserLogin(c, c.Param("login"))
 	if err != nil {
-		c.String(404, "Cannot find user. %s", err)
-	} else {
-		c.JSON(200, user)
+		c.AbortWithError(http.StatusNotFound, fmt.Errorf("Cannot find user. %s", err))
```

Comment:
I'm not sure I agree. A structured JSON response that only has the status code and detail is effectively the same as `c.String(404, "Cannot find user. %s", err)`

I am open to further discussion on the matter, however, from a practical perspective we cannot make this sort of change without modifying the CLI and API as well which currently expect text messages.